bin/ember-template-lint.js

```diff
@@ -20,18 +20,28 @@ async function getSource(filePathToRead) {
   return fs.readFileSync(filePathToRead, { encoding: 'utf8' });
 }
 
-async function lintFile(linter, relativeFilePath, cliOptions) {
-  let resolvedFilePath = path.resolve(relativeFilePath);
-  let filePath = resolvedFilePath === STDIN ? cliOptions.filename || '' : relativeFilePath;
+async function buildLinterOptions(relativeFilePath, cliOptions) {
```

Comment:
This could be exported for unit testing too, no need yet but seems like a nice extraction...